zkevm-circuits/src/super_circuit.rs

```diff
@@ -332,6 +341,31 @@ impl<const MAX_TXS: usize, const MAX_CALLDATA: usize> SuperCircuit<Fr, MAX_TXS,
         let chain_id = block.context.chain_id;
         let tx_circuit = TxCircuit::new(aux_generator, chain_id.as_u64(), txs);
 
+        // NOTE: There is no implemented From<U256> for U64
+        let into_u64 = |val: eth_types::Word| -> U64 {
+            let U256(ref arr) = val;
+            if arr[3] | arr[2] | arr[1] != 0 {
+                panic!("Could not convert block number into U64");
+            }
+            U64([arr[0]])
+        };
```

Comment:
I think this is exact what `U256::as_u64` is doing, so line 360 could be updated to:

```rust
number: block.context.number.as_u64().into(),
```